File: DevFriend_API/DevFriend_API/Entities/Tutorial.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DevFriend_API.Entities
{
    public class Tutorial
    {
        public Guid Id { get; set; }
        public string title { get; set; }
        public string tutorialUrl { get; set; }
        public string iconUrl { get; set; }
        public string description { get; set; }
        public ICollection<Tag> tags { get; set; }
        public Guid UserId { get; set; }

    }
    
}

File: DevFriend-SPA/src/app/_models/tutorial.ts
import { Tag } from './tag';

export class Tutorial {
    id:string;
    title:string;
    tutorialUrl:string;
    iconUrl:string;
    description:string;
    tags:Tag[];
    UserId:string;

    constructor(){
        this.tags = new Array() as Array<Tag>;
    }
}
